.github/workflows/plugin-update.yml

```diff
@@ -1,26 +1,25 @@
 name: Plugin update
-run-name: Update ${{ inputs.repo }} by @${{ github.actor }}
+run-name: Update ${{ inputs.plugin }} to v${{ inputs.version }}
```

Comment:
Don't update the plugin workflows. As it's dispatchable the version on main will be used and I'm not sure if they should be updated.